Cargo.toml

```diff
@@ -19,12 +19,9 @@ numtoa = "0.2"
 [dev-dependencies]
 env_logger = "0.6"
 log = "0.4"
-serialport = { git = "https://gitlab.com/dbrgn/serialport-rs", branch = "embedded", features = ["embedded"], default_features = false }
+serialport = { git = "https://gitlab.com/fkohlgrueber/serialport-rs", branch = "embedded", features = ["embedded"], default_features = false }
```

Comment:
This change is temporary and can be reverted as soon as https://gitlab.com/dbrgn/serialport-rs/-/merge_requests/1 is merged.

Reply:
Thanks! That reminded me of https://gitlab.com/susurrus/serialport-rs/-/merge_requests/76, which I just resurrected.

Reply:
Seems like serialport-rs is seeking a new maintainer (https://gitlab.com/susurrus/serialport-rs/-/issues/111), so I'm not sure if / when your PR will get merged.

Reply:
True. Let's keep your branch for now, we can still decide what to do once embedded-hal 1.0 is released.